cmake/Modules/FindQhull.cmake

```diff
@@ -61,34 +105,35 @@ endif()
 
 get_filename_component(QHULL_LIBRARY_DEBUG_NAME "${QHULL_LIBRARY_DEBUG}" NAME)
 
-if(QHULL_INCLUDE_DIR AND QHULL_LIBRARY)
-
-  # Include directories
-  set(QHULL_INCLUDE_DIRS ${QHULL_INCLUDE_DIR})
-  unset(QHULL_INCLUDE_DIR)
-  mark_as_advanced(QHULL_INCLUDE_DIRS)
-
-  # Libraries
-  set(QHULL_LIBRARIES optimized ${QHULL_LIBRARY} debug ${QHULL_LIBRARY_DEBUG})
-  unset(QHULL_LIBRARY)
-  unset(QHULL_LIBRARY_DEBUG)
-  mark_as_advanced(QHULL_LIBRARIES)
-
-endif()
-
 include(FindPackageHandleStandardArgs)
 find_package_handle_standard_args(Qhull
   FOUND_VAR QHULL_FOUND
-  REQUIRED_VARS QHULL_LIBRARIES QHULL_INCLUDE_DIRS
+  REQUIRED_VARS QHULL_LIBRARY QHULL_INCLUDE_DIR
 )
 
 if(QHULL_FOUND)
   set(HAVE_QHULL ON)
-  if(NOT QHULL_USE_STATIC)
-    add_definitions("-Dqh_QHpointer")
-    if(MSVC)
-      add_definitions("-Dqh_QHpointer_dllimport")
+  add_library(QHULL::QHULL ${QHULL_LIBRARY_TYPE} IMPORTED)
+  set_target_properties(QHULL::QHULL PROPERTIES INTERFACE_INCLUDE_DIRECTORIES "${QHULL_INCLUDE_DIR}")
+  set_property(TARGET QHULL::QHULL APPEND PROPERTY IMPORTED_CONFIGURATIONS "RELEASE")
+  set_target_properties(QHULL::QHULL PROPERTIES IMPORTED_LINK_INTERFACE_LANGUAGES_RELEASE "CXX")
```

Comment:
The language option is the same for both configurations. Can you just set `IMPORTED_LINK_INTERFACE_LANGUAGES` once?

Reply:
Done